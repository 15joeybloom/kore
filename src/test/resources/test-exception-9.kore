[]

module TEST-EXCEPTION-9

  axiom{}
    #X:#S{} []

endmodule []
