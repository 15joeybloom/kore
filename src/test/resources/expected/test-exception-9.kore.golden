Parse error: Failed reading: metaSortConverter: Invalid constructor: '#S'..